### PR TITLE
Add deploy notifications with Slackistrano

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

**Comment by kookster** [comment]
looks good to me.

File: config/deploy.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 lock '~> 3.1'
 
-set :application, 'prx.org-frontend'
-set :repo_url, 'git://github.com/PRX/PRX.org-frontend.git'
+set :application, 'www.prx.org'
```
Comment (farski) [inline config/deploy.rb]: @kookster @chrisrhoden can someone confirm this change isn't breaking?

**Comment by chrisrhoden** [comment]
@farski have you tried a staging deploy?

**Comment by farski** [comment]
No, but I didn't add the hook to staging.

File: config/deploy/production.rb [hunk 2]
```diff
@@ -1,4 +1,14 @@
 role :web, %w{deploy@beta.prx.org}
 set :deploy_to, '/var/www/domains/prx.org/m'
 set :branch, :master
-set :default_env, { path: "/opt/node/current/bin:/opt/python/current/bin:$PATH" }
+set :default_env, { path: '/opt/node/current/bin:/opt/python/current/bin:$PATH' }
+
+set :slack_webhook, -> {
+  webhook = nil
+  on roles(:web) do
+    cmd = 'cat /var/www/domains/prx.org/hal/shared/config/slack_webhook.txt'
```
Comment (chrisrhoden) [inline config/deploy/production.rb]: This should probably be pulling from the deploy location for this app rather than for another app.

**Pull request merged**